src/core/pipeline/accumulators/cached.ts

```diff
@@ -37,7 +37,7 @@ abstract class AbstractCachedAccumulator
 export class SumCachedAccumulator extends AbstractCachedAccumulator {
   declare __cachedValue: number | undefined;
 
-  constructor(definition: Omit<AccumulatorDefinition, 'operator'>) {
+  constructor(definition?: Omit<AccumulatorDefinition, 'operator'>) {
```

Comment:
What happens if the definition is undefined ?

Reply:
It was for some unit tests where the definition isn't useful: but it's probably a better way to pass a mock definition instead of changing the constructor for testing purposes.
I reverted this change and I used a mock definition when needed.